web-app/CKS.Web/Startup.cs

```diff
@@ -51,5 +53,20 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 endpoints.MapControllers();
             });
         }
+
```

Comment:
Whitespace problem: fucntion seems to be out of line from the one above

Reply:
I've fixed it in this branch, but there's some changes to C# formatting in https://github.com/nice-digital/cks-gatsby/pull/24/files so I wouldn't worry too much here